### PR TITLE
subtree merge toggle repo and switch to using quickcache

### PR DESCRIPTION
**Pull request by snopoke**
@emord

### CODE REVIEW

File: corehq/ex-submodules/toggle/management/commands/make_toggle.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+from django.core.management.base import LabelCommand, CommandError
+from ...models import Toggle
+
+class Command(LabelCommand):
```
Comment (dimagimon) [inline corehq/ex-submodules/toggle/management/commands/make_toggle.py]: E302 expected 2 blank lines, found 1

File: corehq/ex-submodules/toggle/management/commands/make_toggle.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    help = "Makes a toggle."
+    args = "<slug> *<users>"
+    label = ""
+
```
Comment (dimagimon) [inline corehq/ex-submodules/toggle/management/commands/make_toggle.py]: W293 blank line contains whitespace

File: corehq/ex-submodules/toggle/management/commands/make_toggle.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    label = ""
+     
+    def handle(self, *args, **options):
+        if len(args) < 1:
+            raise CommandError('Have to specify a toggle slug.')
+        slug = args[0]
+        usernames = list(args[1:])
+        toggle = Toggle(slug=slug, enabled_users=usernames)
+        toggle.save()
+
```
Comment (dimagimon) [inline corehq/ex-submodules/toggle/management/commands/make_toggle.py]: W391 blank line at end of file

**Comment by emord** [comment]
:+1: on :white_check_mark:

**Comment by snopoke** [comment]
:white_check_mark:

**Pull request merged**